.github/workflows/build-docker-image.yml

```diff
@@ -0,0 +1,106 @@
+# To test: gh workflow run 'Build Docker Image' --ref kwannoel/workflow-update-branch
+name: Build Docker Image
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag for the Docker image: vX.Y.Z-<tag>-<commit_sha>'
+        required: true
+        type: string
+        default: 'untagged'
```

Comment:
Will `untagged` appear verbatim in the docker image tag? Is this desired?

Reply:
Yeah I prefer it to always have the same structure.